pilot/pkg/networking/plugin/authn/authentication.go

```diff
@@ -29,7 +29,7 @@ import (
 	authn "istio.io/api/authentication/v1alpha1"
 	authn_filter "istio.io/api/envoy/config/filter/http/authn/v2alpha1"
 	"istio.io/istio/pilot/pkg/model"
-	"istio.io/istio/pilot/pkg/networking/plugin"
+	pluginPkg "istio.io/istio/pilot/pkg/networking/plugin"
```

Comment:
renaming should follow same convention as naming. since this is already violated, at least snakecase i guess to stay consistent.
alternatively, a weasel-y suggestion to avoid stutter would be to call the pkg "plugins" instead (which is not very go either but could be an improvement). or maybe "filter"?

Reply:
Renaming is following the same convention as naming: golang names use camel case. I used snake case originally and my IDE's built in linter complained, actually. I'm happy to use snake case since we use it for other imports (in the same file, even).